modules/preview-react/text/lib/Text.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {createComponent} from '@workday/canvas-kit-react/common';
+// type changes no need
+import {type, CanvasTypeHierarchy, CanvasTypeVariants} from '@workday/canvas-kit-react/tokens';
+import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
+
+export interface TextProps extends BoxProps {
+  /**
+   * Type token level: `title`, `heading`, `body`, `subtext`. Should be provided with `size` prop or it will not apply token.
+   */
+  level?: keyof CanvasTypeHierarchy;
+  /**
+   * Type token size: `large`, `medium`, `small`. Should be provided with `level` prop or it will not apply token.
```

Comment:
If one prop is needed for the other, should we make them required props?

Reply:
No, it shouldn't be required to keep a possibility to use text apart from type tokens. Need to play with this to see if I can make this types tied.

Reply:
i would just add default values to those props and then you don't have to work about it!